Packs/IRISDFIR/Integrations/IRISDFIR/IRISDFIR.yml

```diff
@@ -0,0 +1,79 @@
+category: Case Management
+commonfields:
+  id: IRIS DFIR
+  version: -1
+configuration:
+- display: Server IP or Host Name (e.g., https://192.168.0.1)
+  name: host
+  required: true
+  section: Connect
+  type: 0
+- display: ""
+  displaypassword: API Key for authentication
+  hiddenusername: true
+  name: api_key
+  required: true
+  section: Connect
+  type: 9
+- advanced: true
+  display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  section: Connect
+  type: 8
+- advanced: true
+  display: Use system proxy settings
+  name: proxy
+  required: false
+  section: Connect
+  type: 8
+description: IRIS is a collaborative platform aiming to help incident responders to share technical details during investigations. It's free and open-source.
+display: IRIS DFIR
+name: IRIS DFIR
+script:
+  commands:
+  - arguments: []
+    description: IRIS Command to get the last case information
+    name: iris-get-last-case-id
+    outputs:
+    - contextPath: IRIS.case_soc_id
+      description: "SOC ID ticket case"
+    - contextPath: IRIS.case_id
+      description: "case ID ticket number"
+      type: number
+    - contextPath: IRIS.case_description
+      description: "case description"
+    - contextPath: IRIS.opened_by
+      description: "case opened by"
+    - contextPath: IRIS.owner
+      description: "case owner"
+    - contextPath: IRIS.classification_id
+      description: "case classification ID"
+      type: number
+    - contextPath: IRIS.state_name
+      description: "case state name"
+    - contextPath: IRIS.case_open_date
+      description: "case open date"
+    - contextPath: IRIS.case_name
+      description: "case name"
+    - contextPath: IRIS.client_name
+      description: "case client name"
+    - contextPath: IRIS.classification
+      description: "case classification"
+    - contextPath: IRIS.case_uuid
+      description: "case uuid"
+    - contextPath: IRIS.state_id
+      description: "case state ID"
+    - contextPath: IRIS.access_level
+      description: "case access level"
+  - arguments: []
+    description: Return a list of all IRIS DFIR cases
+    name: iris-get-all-cases
```

Comment:
There aren't any outputs for this command?

Reply:
Added.